.github/assets/setup-latest-rancher.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+url="${url-172.18.0.1.omg.howdoi.website}"
```

Comment:
is this an internal domain resolvable in CI?

Reply:
This will expand to either `$url`, or if not set to '172.18.0.1.omg.howdoi.website' which is a 'magic' DNS domain, that will resolve to 172.18.0.1. 172.18.0.1 is the IP of the first k3d cluster, which exposes port 80 and 443.
As far as I know, we still control 'omg.howdoi.website'. Any other magic DNS is fine, too.